doc/api/dgram.md

```diff
@@ -299,6 +299,23 @@ added: v0.6.9
 Sets or clears the `SO_BROADCAST` socket option.  When set to `true`, UDP
 packets may be sent to a local interface's broadcast address.
 
+### socket.setMulticastInterface(multicastInterface)
+<!-- YAML
+added: v7.0.0
```

Comment:
Instead of `v7.0.0`, please use `REPLACEME`. That value will be replaced during the release.